### ISSUE TITLE
I lost 30.000 EURO BTC on this mybtgwallet.

### ISSUE THREAD

**Issue by Merrtiee** [BTCGPU/BTCGPU#213]
Here's my story, I'm currently very, VERY depressed. I'll leave my mark here.. On 8th of November, I heard about a next fork (Something similair like BCH), on the bitcoin chain. I'm just 17 years old, and I was the proud owner of 4,22 BTC and 4,22 BCH. I had bought this amount when the price was 600$/btc, and invested this out of my own pocket, it was meant for a motorcycle, but I thought I'd give it a shot. Now, few years later, the price for 4,22 was plus-minus 30000 euro. So I was interested in this BTG fork, and looked on their site. The site hosted this wallet: "mybtgwallet.com" It was featured on their website, and it allowed you to import your stuff in to a BTG wallet, it was open source so nothing could go wrong, right? There's no disclaimer on their website, no nothing. So I check my balance, right, and it says proudly 4,22 BTG. I'm happy, wouldn't I be. So I leave it, and the next day I come back and check my BTC wallet, it says ALL my bitcoins have been sent to an EXCHANGE. An freaking exchange, I've not even touched my wallet. (My wallet is blockchain.info). I'm in shock, all my live savings, everything, GONE. I'm trying to contact BTG support team, and I hope this message reaches out to them. Because they were the ones who featured this "scam" wallet on their website as "Trusted". They now removed it, with a disclaimer that they're not responsible, what a surprise. I'm HOPING I get atleast the equal amount to my BTG/BCH or any compensation even if it's not the full amount to make this right, I really don't know what to do, I haven't been outside for 2 days now.. I wasn't expecting this to happen.. Please help me reach to the BTG team, and hopefully solve this..

https://prnt.sc/hcec30 https://blockchain.info/tx/f1c0b71f255370364d3eb6197ca4867cad9fe63ddd90f97cba959d6f4e15d84f

**Comment by frenchgox** [comment]
I'm sorry for your loss.

To any others reading this that haven't tried to use Bitcoin Gold yet:

Send all your BTC to a new wallet before importing private keys into a Bitcoin Gold wallet. There is no reason to risk your BTC at the same time that you risk your BTG.

**Comment by mjoh090** [comment]
@Merrtiee It sounds like you are indeed entitled to full compensation as the you used a wallet endorsed by the bitcoin gold website, before the red box disclaimer, and therefore had a reasonable expectation of trust. 

You are not alone in this. Other people have had this experience, and they are being compensated for their loss. 

I suggest that you get together all the information they need to verify your claims, including proof of timelines, and present it to Martin. Let us know the outcome.

**Comment by lindensixx** [comment]
give him money back u idiots

**Comment by Thecreator1** [comment]
And now it becomes real...
I doubt that you will see any of it back.
It was very naive to use a Private key with balance on 3 chains.

**Comment by casaroto** [comment]
My btg wallet was officially on btg website, should have been a red alert on that site right after people started relating steling of coins.
Btg has lost a big chunk of credibility and doesn't seem to care.

**Comment by Merrtiee** [comment]
Surely you can't blame me 100% for it, it was marked as trustworthy on their official site.
@Thecreator1

**Comment by Thecreator1** [comment]
Yes well it's your own fault.
Question everything, trust nobody.
Had you transferred the BTC to another cold wallet then they would have had 0 BTC to rob from you.
I had a lot of paper wallets and i had to work them one by one to prevent what happened to you.
You can not be careful enough with these things.

**Comment by Stephen2929** [comment]
https://www.reddit.com/r/btc/comments/7dsmvd/my_analysis_of_the_1_million_usd_mybtgwalletcom/

**Comment by mjoh090** [comment]
@Thecreator1  You did the prudent thing by not assuming trust, protected yourself, and consequently avoided loss. 

However, the BTCGPU team did take responsibility upon themselves for the action of the wallets they advertised on their website. The red redbox disclaimer was not there from the beginning - its a recent addition. Prior to the explicit disclaimer, they effectively endorsed those wallets that users reasonably interpreted as a signal of trustworthiness.  

Consequently, there was a reasonable expectation of trust by users of that scam wallet. 

To BTCGPU team's credit, they have  recognised their culpability in this by compensating those who lost money using the scam wallet advertised on the bitcoin gold website prior to adding the explicit disclaimer on the front page of their website.

I think that it is irresponsible for the the bitcoin gold team to be advertising wallets on their website, even with the disclaimer, without first ensuring that they are trustworthy. My view is that the red box disclaimer is a lazy cop-out. They should either be certain that the wallets are safe, or not advertise them at all.

**Comment by Thecreator1** [comment]
Yes well i did however lost some funds that i gained while mining BTG in the first days.
These funds disappeared when they reset the gold pool.
I have submitted the form that was linked on twitter with the request to 'make some noise' when the funds were compensated, but as of yet nothing ever came of it and to be honest i do not expect anything from it, so i consider these funds lost.
One has to realize that we are dealing with cryptocurrencies here and that there is no external party that you can turn to whenever something goes wrong, this is just the nature of the beast and it shows time and time again that people have to take their own responsibilities, because gone means gone and can usually not be recovered.

**Comment by evadogstar** [comment]
emotions with bitcoin gold: facepalm

**Comment by evadogstar** [comment]
Lol, my mined BTG gold to address generated with this walled also was stolen. 
BTG GOLD totally scam and crap. Must be closed.

**Comment by Bourne-ID** [comment]
I'm sorry, am I reading this right:

> To BTCGPU team's credit, they have **recognised their culpability in this by compensating those who lost money** using the scam wallet advertised on the bitcoin gold website prior to adding the explicit disclaimer on the front page of their website.

Past tense, IE the devs have paid out? I struggle to see this as it's open to abuse. Could you provide proof of this please. 

In response to:

> think that it is irresponsible for the the bitcoin gold team to be advertising wallets on their website, even with the disclaimer, without first ensuring that they are trustworthy.

Couple of questions:

1) How can you test if a site is trustworthy when there was no mining and transactions took place prior to launch?
 
2) Even if a wallet site is tested, because it's not owned by the BTG team, what's to stop the wallet site from taking all the private keys entered and using them for their own gains after a week? Or month? Or a year? Answer is nothing, so 'ensuring trustworthiness', like every online wallet, is based on reputation, and the decision to give the wallet/private key is down to the owner of said key, in my opinion.

**Comment by evadogstar** [comment]
@Bourne-ID Do you read my situation? There are NO OTHER ways to get G address to start mining. I get it from BTGold website. It contain link to MySCAMwallet. I generate G address and lose little amount of mined coins. FULL SCAM

**Comment by Bourne-ID** [comment]
@evadogstar I did, but I wasn't addressing you as you comment was unconstructive. I was quoting @mjoh090 comments with queries. I'm sorry you got scammed, but that's not the point of my post.

**Comment by evadogstar** [comment]
This shit coin must die. The scammer stole coins formed after the launch. I do not want to support scammers.

**Comment by evadogstar** [comment]
Thanks "devs" or scammers that you provide us scammy G addresses when coin launch.

**Comment by evadogstar** [comment]
I ask MANY times (proof https://github.com/BTCGPU/BTCGPU/issues/164 ) for valid G address BEFORE launch. Devs ignore my requests and what? SCAM!

**Comment by mjoh090** [comment]
@Bourne-ID With regard to compensation being paid  - you are correct. I do not know for sure that this is happening. I have seen indications on the slack channel that this happening, but that of course is not proof. I don't know if the slack channel accounts confirming payment represent genuine people who have been compensated for a genuine loss. 

Your remaining points are also valid. 

Although, I  imagine a level of trust could be assumed if a 'wallet site' has a history and reputation of producing reputable wallets for other coins. There may well be ways to improve trustworthiness and filter out potential scammers - maybe requiring the wallet producers to verifiably make themselves known (and therefore accountable) as a condition of endorsement of their wallets on the website? 

The fact remains however, that for a time, the BTCGPU team advertised wallets without an explicit disclaimer, the absence of which implicitly endorsed those wallets providing a reasonable expectation of trust, and I think that is where the test for responsibility lies - whether or not it was reasonable for people to trust the wallets implicitly endorsed on the bitcoin gold website. I think the answer is yes. 

The BTCGPU team inadvertently endorsed a scam wallet and in so doing made a mistake. This mistake resulted in loss suffered by others. Their mistake, their responsibility.

**Comment by francois007** [comment]
Another big idiot who did not move all bitcoin into a new wallet before using the private key to have the bitcoin gold.

**Comment by Thecreator1** [comment]
Well mjoh090 i just checked the address that i had put in the compensation form and there is nothing.
I guess they are only compensating close friends, do we know you? ah yes here you have some... Who are you ? We don't know you, go away.

**Comment by mjoh090** [comment]
@Thecreator1 Do we know you? Should you go away? Of course not. You have as much right to be here expressing your views as anyone else.

You indicate that you have lost money, and have not yet been compensated. The content of my post advocates for compensation of those who have lost money. Your comment therefore makes no sense.

**Comment by maslovsa** [comment]
Guys,
Is there any trusted ways to collect BTG from my seed or private keys (Electrum).
I read a lot of such scams stories while I was searching the answer.

**Comment by evadogstar** [comment]
@maslovsa No. Looks like devs made the scamcoin only to HYIP quick money. They do not think on long term. They do not give us any wallets on the mainnet start (only full node) and SCAM wallet that stole our mined coins.

**Comment by charleshugh84** [comment]
Would you shut the fk up? You are just whining here like little girls. Why you have entered the key that is said everywhere to be kept completely secured and not to be inputed anywhere manually in a site on the web? When you cannot keep your money safe, why you want to be given more. Learn how to control your money. If the coin was a scam none of the exchanges and wallets would support it (You would not see Bittrex, YoBit, Trezor, Ledger on the main page). So get your shts together and do something useful like this - http://lmgtfy.com/?q=How+to+keep+my+bitcoins+safe

**Comment by charleshugh84** [comment]
All your fckin greed brought you here, because you wanted to make some fast money, now you gonna suffer because of your stupidity.

**Comment by evadogstar** [comment]
@charleshugh84 Are you stupid? We use the way to get G address to mine on mainnet start provided by DEVs. Now, we lose mined coins, because wallet was SCAM. There was NO OTHER way to get G address to provide it for mining pool. proof https://github.com/BTCGPU/BTCGPU/issues/164

**Comment by charleshugh84** [comment]
what is the first answer --- 
btgwallet.online
This is available to be run locally offline.

**Comment by Merrtiee** [comment]
I've NEVER claimed that this new coin is a scam. Honestly ALL I want is my 4.228(currently worth 35k USD) BTC back, and you can keep the BTG. I don't even care. Also I wasn't greedy at all, all I wanted was to check my balance. @charleshugh84 

As @mjoh090 said earlier, we have the right for a refund. I lost all my money I had on this stupid thing, and I can't stop crying about it.

**Comment by evadogstar** [comment]
@charleshugh84 First answer was corrected ~5 hours ago. When the mainnet started it contain SCAM wallet as suggestion and more.. the question ignored by devs at ALL treads. Therefor, I created this special thread to get more attention to this problem, because I still (in those days) assumed that this is a SCAM wallet, but what? FULL IGNORED by devs and this wallet was on the bitcoingold.org website provided as trusted before one hour of mainnet launch. I wanted to participate on mainnet launch and support the coin, but I needed G address and this was the only way to get it.

**Comment by zarabiajkase** [comment]
What do you thinking about freewallet.org on android? They have got many wallets for altcoins whith BTG too.

**Comment by charleshugh84** [comment]
@evadogstar how convenient. You say that all your private keys are stolen. Where to send the money that you are whining for? On the same addresses to be stolen again? Think with your heads! 
Do you think that some of the persons who lost thousands of funds can be the scammer, and uses you to get more money? Man up and learn how to use cryptocurrencies. This is like to have given the scammer your credit card and then to blame the bank.

**Comment by evadogstar** [comment]
@charleshugh84 Again, I do not upload my bitcoin private keys to any resources. I'm only generate new key&G-address to start mining and support the coin on mainnet launch. It was only one way to get G address to start mining with: get it from scammy service provided by devs.

**Comment by charleshugh84** [comment]
Few days ago there was a scam on blokchain.info advertised in google. Go and blame google for showing ads of a scam. 
Man up and start using proper wallets. The fact that you are ignorant put you in that condition and you can blame only yourself. This is a witch hunt and you must accept your blame. Why I and so many other people didn't got scammed, because we don't share our private keys. This is why they are called private.

**Comment by Merrtiee** [comment]
@charleshugh84 You're being a dick, and honestly you should just keep out of it? This was a call to the devs to get some support, this wasn't a call of dicks.

**Comment by evadogstar** [comment]
@charleshugh84 "proper wallets" ? Again, it was only one "proper wallets" provided by devs on mainnet launch: https://github.com/BTCGPU/BTCGPU/issues/164 
What you can not understand?

**Comment by charleshugh84** [comment]
@evadogstar to have an address you could just have downloaded BTG Core wallet - this is the only wallet provided by the devs. 500 MB in pruned mode. I'm using it. I also use the Bitcoin Core wallet again in pruned mode as well as Ethereum Go wallet. I have no problems.

**Comment by charleshugh84** [comment]
@Merrtiee language! You are 17. 
I was scammed the first day I got into crypto. I lost 7 bitcoins that I have bought on the previous day. I have downloaded an Android wallet created an account and sent all my money. The next morning I had 0.000001 balance. Do I have to blame the PlayStore, or the site I've bought the coins, or the phone? This world is not for the fainthearted, you are going to the casino with each transaction in the crypto world. When many people win, many other - lose. 
Man up and start working to find the one that have taken the money. Write to exchanges, wallet, everything. This is not Z-Cash, nothing is anonymous, exchanges can catch the guy, they need verification for trading. 
To make this topic productive start posting the addresses of the scammer.

**Comment by Merrtiee** [comment]
@charleshugh84 I'm sorry for my language, I'm just so frustrated, I'm sure you'll understand..
We've already written to ALL exchanges we can think of.

**Comment by charleshugh84** [comment]
It would be useful to post them here also. I will send mail also and there is no full list anywhere.

**Comment by evadogstar** [comment]
@charleshugh84 And ask you ones again: are you stupid? On the mainnet launch I want to participate on this historical moment and support this coin... ( scam coin :( ). It was NOT been possible to download and install BTG Core wallet before and after mainnet launch! Proof: 
https://github.com/BTCGPU/BTCGPU/issues/172 
https://github.com/BTCGPU/BTCGPU/issues/176
And other issues. 
One again: 
1) It was not  possible to install it to get G address and start mining!
2) I use only one possible way: suggested web wallet from devs to get G address and start mining. 
All little amount of mined was stolen. 
Ones again SCAM

**Comment by evadogstar** [comment]
@charleshugh84 And more. Now it's very risky to install this wallet :) I do not want to risk so much to install some "open source" from such developers :) 

I am still confident that this is the best solution to use web wallet for some kind like this scam coins, because it is safer for my machine VS run some scam code from unknowns scam developers.

**Comment by charleshugh84** [comment]
Bullshit. 

I have installed the core wallet and have taken an address before it was synched. This is not excuse.

I think that with all that hate you are one of the thief and you want to turn on the people so that can insist on refund and then you to take the money again.  

I have told you how to help yourself, just post the addresses that have received the stolen money. That way they will be marked and cannot be used. You are dumb as fck and even more greedy. Only wait the others to fix the situation and hate by that time. I hope you never receive a single crypto coin again, because you don't deserve it.

**Comment by JokerCatz** [comment]
Hi all , I'm not dev team , but I'm a developer for a exchange site , I think the BTG group is not ready , like hasn't a full explorer or some wallet service , but ... if you lost some coin , I think it your fail , not official team , the code is open source , everybody can be review ... so , if you lost some coin , you must import private key to another service "or" you trust a stupid explorer or wallet service for it not show your right balance ... so your can wait this coin is ready ( good env of explorer or wallet services ) or you are a developer , or reply txid / wallet / block height to help another guy to trace it , I think it will be okay for this topic / issue ...

**Comment by maslovsa** [comment]
Colleagues
BTW Why collect BTG from private keys or seed?

**Comment by evadogstar** [comment]
@charleshugh84 It will not help. Look at this thread some address already posted https://github.com/BTCGPU/BTCGPU/issues/194 

BTW Scammer use mixer

Open your eyes and read that many ppl lose there mined coins who mine on the mainnet launch. Not only me. I'm already notified about groupof ppl at "slack" or "sluck". They already are ready to open issue on police and cyber crime organizations about that all.

**Comment by evadogstar** [comment]
Give me one seconds :) I know what is that feature that not work until main net launch! :)

**Comment by emre9712** [comment]
Here are the adresses which the scammer used when stealing my coins 

GNr19CrZ3EzYQVZ6DLJXfLtTxWu9RvhokG

GfVhxiz7CEgGeTkrxeyPbpXCgsHcPWb4Bh

GRvhh6sTUrgjsqpxTm7KDCU9RvD9MstgJe

GbtGnXxvcAkXVQp8WTkTtu8jhLZWgqkPXW

GbwA3Q6zgRPDuznhQzbKkfT511wMc9bRqT

GT6oPUqB76LcTuttZe4AU5HAbheJJZzvKB

Geq1aXh8vbjEsYqNANTFrbzfwT3gzGXbhe

GftWLQG9Y5GUEyafihbhrY2eXPWFBihFzD

**Comment by JokerCatz** [comment]
first , we need proof , use math / code , not dirty word ...... all transaction need to sign and broadcast it => add to chain , and break signature system is too difficult (it can be proof from math) , so only you leak your private key ( maybe gen from seed ) , your coin are lost ... the chain data can't be fake , it can be verify by everyone , and miner will be verify it many times ...

if we don't need proof ... the ssl / https / telegram ... all internet security will be shit ... and leak everything ... not only your coin , okay?

and my site customer is very funny , leak password / leak email / lost mobile / hacking / virus / side-channel attack , even this is happened

https://www.symantec.com/security_response/writeup.jsp?docid=2016-020216-4204-99

so your guys , you really think dev team steal your coin ? or you find wrong place ?

**Comment by JokerCatz** [comment]
@evadogstar @emre9712 

I think your condition is official to much to trust unconfirmed third party side ... but I think it too late ... and dev team can't do anything for this , because you can't proof it is your wallet and it be steal not you send it right ? so ... like my funny customer , I only feel sorry , but I can't do everything ... or any help , if I'm dev team , it too

**Comment by Merrtiee** [comment]
@JokerCatz Something like a KYC can be done. We can give our privatekeys again (as they're empty anyways) to proof that we really lost the money.

**Comment by JokerCatz** [comment]
@Merrtiee

I say again ... [ because you can't proof it is your wallet and it be steal not you send it right ? ]

sorry , it nobody can , because all cyber currency wallet / transaction is anonymous ... so I can steal a wallet say it my wallet ... right ?

**Comment by evadogstar** [comment]
>>first , we need proof ,

@JokerCatz , NO, first, we need proof that it's not you or some other dev stolen our coins, because here is already big community that lose money after devs actions.

http://forklog.net/bitcoin-gold-users-lose-over-2-5m-and-accuse-the-team-of-scam/

So, where is the proof that all this affected persons not scammed by you or other devs?

**Comment by evadogstar** [comment]
@Merriee We does not need to proof somethink, because 90% of ppls who start mining lose money. Then it was discovered that provided by devs wallet was SCAM - It's FACT. 

http://forklog.net/bitcoin-gold-users-lose-over-2-5m-and-accuse-the-team-of-scam/

It could be a planned action from devs against community. 

You are not alone! If you will be alone who say that your coins stolen. Okey, I can be scammer. But when all ppl say that it was scammed by devs. Make conclusion.

**Comment by JokerCatz** [comment]
@evadogstar no ... it not fact , it a blog write by today ...

[ Mybtgwallet was the only way to generate a BTG address for miners ]

it I say , it the third party not official and your foolish , did not know your wallet must gen by self and can be convert from bitcoin wallet ... so your condition very like I say " don't share your private key in public area ... " , and official too trust third party service ( I don't want to loop it ... )

so no body can help you ... I am sorry.

**Comment by iiqqrs** [comment]
I too was stolen from with that damn wallet. I transferred all my btc out before like any sane person would have. But still, what a shady operation.

**Comment by JokerCatz** [comment]
@iiqqrs like my customer lost coins , I just feel sad , but I can't do anything ... even I am a developer ... it cyber currency rule ...

MT Gox / Bitfinex / Bithumb ...... all follow the rules ...

**Comment by Merrtiee** [comment]
@JokerCatz BTG will always have blood on their name unless something gets water to wash it off.. They can make this right, I'm sure they can..

**Comment by JokerCatz** [comment]
@Merrtiee no they can't , if they fork again , the miner will be all lost , it will be very bad for env , like ETH & ETH classic , and market price will be fail down , and your money is never back too ...

and like I say , no body can be proof wallet / transaction is steal ... ETH can be proof because smart contract , but bitcoin like is not ...

**Comment by bitcoin-gold** [comment]
Who suffered from the actions of devs join community without censorship to discuss the issue and we will decide what to do with this https://github.com/bitcoin-gold/community

**Comment by h4x3rotab** [comment]
@bitcoin-gold There's no censorship, only GitHub abusing. There are forums and sub-reddits. GitHub is about development.

**Comment by Merrtiee** [comment]
@h4x3rotab I'm sorry for abusing, but you surely understand my frustration and of many others..

**Comment by francois007** [comment]
This is a space for developers and not a forum for idiots!
Please close this issue.

**Comment by Merrtiee** [comment]
@francois007 you sound very mature..

**Comment by francois007** [comment]
@Merrtiee and you are a herd of idiots

**Comment by mjoh090** [comment]
@h4x3rotab  

While Github is indeed a place for development, this Github repository is the place to get the 'official' BTCGPU client, and new miners may well make a bee line here oblivious to the issues being discussed on forums on other platforms. 

This issues section therefore may be the first encounter new miners have with issues that go beyond coding, and enables them to make more informed decision about how they wish to engage with BTCGPU. 

These others non-coding issues may well be more important than the coding issues. If this coin was for example a scam with perfect code, would you think it right to allow the devs to justify an attempt to shut down the issues raised by users on the basis that the issues raised are not code related, and in so doing are allowed to dupe new miners ( who may be unaware of the shared observations and concerns raised by individuals in forums on other platforms ) into enabling the scam? 

While the above is an extreme example, it remains an example where a case could made for justifying the use of the issues section on Github as a tool to raise awareness, because it would be in the public interest to do so.

While Github is a place for development, most other development projects probably don’t have quite the same level of concern for the public interest as GTCGPU does, as there are a large number of people who have bought into this project made up of investors,  individual miners and pool operators. 

People have spent, time, resources and a reasonable expectation of trust on this project, and have lost real money. These same people may well feel a duty care to  maximise awareness of the persistent issues and concerns around this project. Since all ‘roads’ to the BTCGPU project leads to the residence of the official client, it makes this area a reasonable staging ground for raising awareness of important non-coding related issues, and alerting those looking to invest time and resources into this project.  

I think there is a case to be made here that it is in the public interest to allow this issues section to include a wider range of issues beyond just code.

**Comment by Thecreator1** [comment]
> we have the right for a refund. I lost all my money I had on this stupid thing, and I can't stop crying about it.

@Merrtiee 
You can't be serious about this.
This would be possible with a damaged banknote, but not with cryptocoins, it's clear to me that you do not understand this technology.
There is nobody you can turn to.
It was an expensive lesson for you, now take your loss and make sure it doesn't happen again.
Seeing you write that you think you are obliged to a refund immediately gives me the impression that you haven't learned enough, be careful...
Once more: There is no one you can turn to.
That is the nature of the beast.

**Comment by projectdark8** [comment]
Always use the CLI wallet that is included with the blockchain of the specific coin if you can. Yes this eats up a lot of space, but you cannot trust any open source program on the internet. Welcome to the world of crypto man, sorry for your loss.

**Comment by Ayms** [comment]
Please see https://github.com/BTCGPU/BTCGPU/issues/226#issuecomment-346672481

**Comment by evadogstar** [comment]
you do not see any contradictions, guys? @projectdark8 @charleshugh84 

1)
> use the CLI wallet

2) 
> you cannot trust any open source program on the internet.

The fact that you write does not bother you? Ones again: you say that you can not trust any open source program. But on the same comment you wrote that we must trust some scammy code from unknown developers that already got our 0,5% on the pool issue, then provide us with scammy online wallet, then electronGold scam. Sure?:)) We must trust and install this wallet? :)

**Comment by Ayms** [comment]
Indeed, open source does not mean that you can trust it, it's quasi impossible to review a huge program like bitcoin core (or Tor for another example) and all dependencies, nobody never check it,

That's why I made https://github.com/Ayms/bitcoin-transactions, that's not a stupid wallet sw, see https://github.com/BTCGPU/BTCGPU/issues/226#issuecomment-346798767

**Comment by evadogstar** [comment]
@Ayms Yes. we are already known about all scam projects on bitcoin gold.

**Comment by Ayms** [comment]
I tried to read this long thread again, that's amazing, as well as all of what I can read in other places

People think that they are transfering coins from "a wallet to another", then the coins are supposedly protected, they are just transfering coins from a wallet address to another wallet address

Don't people understand that they just pay fees for nothing doing this and that the only important thing is the private key corresponding to a certain address which can be used with different wallets without transfering anything? Even if not recommended of course, I personnaly will never use a sw wallet

Because what is the use of wallets btw?

Get info about something that you can easily get from a blockchain explorer?

Provide you converted addresses? (the conversion is trivial when you know it, not for common people I agree, but you should know that the addresses are the very same between networks)

Then compute your transactions using your private key that you gave them, like giving the key of your home to anybody?

Using wallets is the easy way solution and extremely dangerous, that's the very intent of https://github.com/Ayms/bitcoin-transactions and https://github.com/Ayms/bitcoin-wallets

Of course it's less easy/convenient than wallets, you must do some work, but at least you know what you are doing

**Comment by Technophant** [comment]
100,000 premined coins.  Free essentially.  At least 1,800 stolen BTG, maybe more stolen, let's say it's 2% of that.   It's reasonable to ask that at least the stolen BTG be returned.  You keep it all as a reward for a job well done, but this is disaster.  Nobody will trust this team if they don't at least refund the stolen BTG.

**Comment by seangeleno** [comment]
What a great life lesson, espensive, but invaluable. “There's an old saying in Tennessee, I know it's in Texas, probably in Tennessee that says, 'Fool me once, shame on ... shame on you. Fool me... You can't get fooled again“ a quote that’s almost older than you!

**Comment by weltreisendertj** [comment]
I read all this stuff here and thinking about why the btg team doesn´t write a step by step tutorial: How I can claim my btg-gold from btc core wallet to the btg gold wallet? Is it so difficult or have they fear that all people sell it? I can only find conami tutorials or SCAMS. For people without a android system it is difficult to find out how it works.

**Comment by Ayms** [comment]
See https://github.com/Ayms/bitcoin-transactions#addresses-and-getting-free-coins

**Comment by weltreisendertj** [comment]
This is absolutly not usefull for me! I´m not familiar with the btc core wallet command line and also not with the btg full node command line.

Now I should use another tool to the the BTG from the chain? Why I should trust this tool? And why I should learning all this address format converting stuff? All of this is absulotly user unfriendly!

And my question again: Why the btg-gold team is not writting a official tutorial on there official page twitter or facebook account?

This is from you link: "However, if you want to move your bitcoins "from bitcoin core to bitcoin gold" (which as explained above means nothing) or from "bitcoin core to a bitcoin gold exchange", you can just use the create command:

node tx.js BTG create prevtx= prevaddr= prevamount= previndex= privkey=privkey addr= fees= amount=(optional)

where prevxxx refers very exactly to the same that you can see in a bitcoin core explorer like https://blockchain.info before block 491407 (same transaction id, same address, same amount, same index) and privkey is the private key corresponding to your bitcoin core address"

???

Please help all the people which have not a deeper understanding of the Bitcoin BlockChain!

We need a step by step tutorial. Maybe with less technial explanations.

**Comment by Ayms** [comment]
I am not part of the gold team, the interest of this tool is that you don't need to sync a node and/or a wallet to build transactions, and you don't need to run the bitcoin sw, see https://github.com/BTCGPU/BTCGPU/issues/238#issuecomment-350291549 too

Another interest is to manage everything by yourself, cli for example does "what it likes" and can send amounts in multiple addresses without asking you, unlocking addresses without asking you, etc, then at the end your wallet becomes a mess and making a transaction can involve many inputs, the transactions are huge and therefore your are paying many fees for nothing

I am trying to explain things but nothing is simple and apparently it's not enough

What I mean by "however..." is that you can use the very same info that you can see in a bitcoin core explorer to make a gold transaction with the tool and send your btgs to another address (exchange for example)

Example: you have 1 BTC in transaction 1234 on address 1ABC on bitcoin core before block 491407, you can send the corresponding 1 BTG to a gold address Gxyz (one of yours, one of an exchange) doing

node tx.js BTG prevtx=1234 prevaddr=1ABC prevamount=1 previndex=x privkey=(1ABC privkey) addr=Gxyz fees=0.00001001

or even to another bitcoin address of yours 1DEF which will correspond to a gold address GDEF if you don't know how to convert for now doing 

node tx.js BTG prevtx=1234 prevaddr=1ABC prevamount=1 previndex=x privkey=(1ABC privkey) addr=1DEF fees=0.00001001

Then in both cases the tool will output the corresponding gold G address both for 1ABC and 1DEF, and you will get 1 BTG minus the fees on that address, because the addresses are just the same, the only (VERY) important thing is that you have the corresponding private key (for non P2SH addresses that you must not use for now, see again the issue referenced above), so only use addresses that you master (so don't use example 2 and 1DEF if 1DEF is an address you have in an exchange for example, get a G address from the exchange associated to your gold account)

And then you can check with a gold explorer that your transaction occured on the gold network

I don't know how to do more simple but can take suggestions, and why you can trust this tool is because it's a few lines of code that are esay to review, and the output tells you everything about what you are doing before you send the transaction, so you can check it

You can test addresses conversion using https://github.com/Ayms/bitcoin-wallets

**Comment by marcomarsala** [comment]
The scammer may be identifiable. IP address of who uploaded the website via FTP is logged. Domain name is registered to someone. IP of who bought the domain name is logged. He payed the registration with a *credit card* belonging to someone. Google Analytics ID used in the site is connected to a Gmail; every IP accessing any connected Gmail service is logged. Every IP is linked to a residential internet access bill.

I suggest to report these facts to the police, they will request IPs and credit card evidences to Google and to the provider where the domain was bought. Then, they will identify the network provider who the IP belong, and they will request name and surname of who pay the bills.

**Comment by h4x3rotab** [comment]
@mariodian We have done all the things you mentioned and encourage all the victims to do the same. Sadly police didn't do anything and without their help the service providers won't do anything.

**Comment by marcomarsala** [comment]
How do you know the police didn’t do anything? Did you place a formal complaint? Don’t go to the police office, place a written complaint in the prosecutor’s office located in your local court. You have the right to add to your complaint the request to be formally notified if the complaint is archived, so you will know the motivation and you can take further actions. Don’t know how it works in your country, in mine this is what I would do. If in doubt about procedures, consult a lawyer. I also found various specialized law firms on Google.

MM
________________________________

**Comment by Brennmyers-Geurin** [comment]
I quit my job at Seven Eleven after realizing I was making more money in my sleep trading cryptocurrency. Within three months of trading Bitcoin, I made $270,000 by buying and selling at the right times and to the right people. This success allowed me to network with some of the biggest traders, and within six months, I was trading up to $800,000.I was fully invested in my new career, spending countless hours analyzing the market and making strategic trades. However, my success attracted the wrong kind of attention. One of my close associates, someone I trusted implicitly, initiated a trade with me. Unbeknownst to me, he had enabled a phishing scam that locked me out of my email and trading accounts. The sense of betrayal was overwhelming. This associate had been someone I considered a friend, and his actions left me vulnerable and distressed. My hard-earned assets were suddenly at risk, and I felt helpless. In my desperation, I remembered hearing about the Daniel Meuli Web Recovery Team through Telegram. (at) Danielmeuli I contacted them immediately, explaining my situation. Their response was swift and professional. They assured me they could help and quickly began working on my case. The team used advanced security measures to recover my email and secure my trading accounts. Their expertise and dedication were evident throughout the process. The Daniel Meuli Web Recovery Team uncovered the extent of the scam, confirming my associate's involvement. Armed with this evidence, I reported the matter to the authorities. The law acted decisively, and my associate faced severe penalties. He settled the matter to avoid further legal consequences, returning a significant portion of the stolen funds. This experience was a harsh lesson in trust and the importance of securing personal information. The betrayal stung deeply, but the swift action of the Daniel Meuli Web Recovery Team helped me recover my assets and restore my peace of mind. Their professionalism, prompt response, and effective recovery efforts make them the best in the field. Thanks to their help, I was able to bounce back from this setback and continue my successful journey in cryptocurrency trading. 
What's App  +393 5120 135 28
![WhatsApp Image 2024-05-31 at 04 32 49_ab36aa0d](https://github.com/BTCGPU/BTCGPU/assets/173321383/7649e925-9338-4315-9b78-a8e7a8ffb24d)

**Comment by mjoh090** [comment]
@Brennmyers-Geurin 😂 The audience you mean to target is on TikTok, not GitHub.

**Comment by MandolinaGomez** [comment]
I experienced a distressing setback on January 5th, 2024, when I inadvertently clicked on an airdrop link in my Trust wallet, resulting in the theft of 10.7 Ethereum from my account. The shock and frustration were overwhelming, as I watched my hard-earned digital assets vanish instantly. Unsure of where to turn for help, a female friend from Russia recommended Trust Geeks Hack Expert as a potential solution. Skeptical at first, I hesitated, fearing that Trust Geeks Hack Expert might be just another online scam promising recovery services only to disappear after taking payment. However, after careful consideration and desperate circumstances, I decided to give it a try. My initial contact with Trust Geeks Hack Expert was met with cautious optimism. I explained my situation and shared my skepticism about online recovery services. To my surprise, their team responded promptly with empathy and professionalism. They reassured me of their legitimacy and outlined a clear strategy to retrieve my stolen Ethereum. Despite my lingering doubts, I decided to proceed, hoping against hope that Trust Geeks Hack Expert could deliver on their promise. True to their word, Trust Geeks Hack Expert exceeded my expectations. Within an astonishingly short span of less than 36 hours, they successfully recovered the entire 10.7 Ethereum that had been stolen from my Trust wallet. The speed and efficiency with which they handled my case were nothing short of remarkable. It was a moment of immense relief and gratitude, knowing I had regained what seemed lost forever. The process of working with Trust Geeks Hack Expert was seamless and transparent. They maintained constant communication, keeping me informed of every step they took to track and retrieve my stolen funds. Their expertise in digital asset recovery and familiarity with blockchain technology were evident throughout the recovery process. They navigated the complexities of cryptocurrency transactions with precision and dedication, leaving no stone unturned in their pursuit of justice. Reflecting on my experience, I cannot overstate the significance of Trust Geeks Hack Expert role in restoring my financial security and peace of mind. They not only recovered my stolen Ethereum but also restored my faith in the possibility of recourse against cyber theft. Their commitment to client satisfaction and their unwavering support in times of crisis distinguish them as a trusted ally in the realm of digital asset recovery. I am profoundly grateful to Trust Geeks Hack Expert for their exceptional service and their genuine concern for their client's well-being. They are more than just a recovery service; they are a beacon of hope for individuals like myself who have fallen victim to cybercrime. Their ability to deliver results swiftly and effectively sets them apart in an industry fraught with uncertainty and skepticism. To anyone who finds themselves in a similar predicament—whether it's stolen Cryptocurrencies, unauthorized transactions, or other digital asset losses—I wholeheartedly recommend Trust Geeks Hack Expert. Their track record of success, coupled with their commitment to integrity and client satisfaction, makes them the go-to solution for navigating the complexities of digital fraud recovery.

 CONTACT SERVICE  
Email::  trustgeekshackexpert [At] fastservice [Dot] com
Telegram ID : Trust_geeks_hacke_xpert
Email: info @ trustgeekshackexpert . com
![WhatsApp Image 2024-06-01 at 14 20 19_2e660a7a](https://github.com/BTCGPU/BTCGPU/assets/173854003/938914e8-e471-49e4-aaf3-5dab1ab844c5)

**Comment by Thecreator1** [comment]
You’re so full of shite and you wish you had any eth but your a poor broke ass motherfucker trying to scam other humans and its so fucking unfortunate its a pity you decided to roll like this its so fucking translucent boi go stroke your stinking penis with your dirty black hands because its the only thing your good at and thats not going to change any time soon so go on and leave us alone we are not going to send you anything so fuck off boi

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by marssystems** [comment]
This is a scam - beware !!

**Comment by CharlotteJennifer** [comment]
RECOVER YOUR SCAMMED MONEY WITH THE HELP OF INTELLIGENCE CYBER WIZARD.

Hi. Please be wise, do not make the same mistake I had made in the past, I was a victim of bitcoin scam, I saw a glamorous review showering praises and marketing an investment firm, I reached out to them on what their contracts are, and I invested $55,000, which I was promised to get my first 15% profit in weeks, when it's time to get my profits, I got to know the company was bogus, they kept asking me to invest more and I ran out of patience then requested to have my money back, they refused to answer nor refund my funds, not until a friend of mine introduced me to a recovery company, so I reached out and after tabling my complaints, they were swift to action and within 2/³ hours I got back my funds with the due profit. I couldn't contain the joy in me. I urge you guys to reach out to INTELLIGENCE CYBER WIZARD ASSET RECOVERY SERVICE on their support contact details below.

Email: intelligencecyberwizard@gmail.com

Mail: intelligencecyberwizard@cyber-wizard.com 

Mail: intelligencecyberwizard@mail.com

What's App/Texts (PLUS) 1 (219) 424-7566

**Comment by Thecreator1** [comment]
Scammer 😡

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by teyas131** [comment]
<html>
<body>
<div class="gmail-adn gmail-ads" style="border-left: none; padding: 0px; display: flex; font-family: &quot;Google Sans&quot;, Roboto, RobotoDraft, Helvetica, Arial, sans-serif; font-size: medium;"><div class="gmail-gs" style="margin: 0px; min-width: 0px; padding: 0px 0px 20px; width: auto;"><div class="gmail-"><div id="gmail-:g2" class="gmail-ii gmail-gt" style="direction: ltr; margin: 8px 0px 0px; padding: 0px; font-size: 0.875rem; overflow-x: hidden;"><div id="gmail-:g3" class="gmail-a3s gmail-aiL" style="direction: ltr; font-variant-numeric: normal; font-variant-east-asian: normal; font-variant-alternates: normal; font-size-adjust: none; font-kerning: auto; font-optical-sizing: auto; font-feature-settings: normal; font-variation-settings: normal; font-variant-position: normal; font-variant-emoji: normal; font-stretch: normal; font-size: small; line-height: 1.5; font-family: Arial, Helvetica, sans-serif; overflow: auto hidden;"><div>In one notable case, a victim of a hacked cryptocurrency exchange turned to Infinite Digital Recovery for help.<br></div><div><div class="gmail_quote"><div dir="auto"><div>Despite the complex nature of the incident, the team at Infiniti Digital Recovery diligently investigated the blockchain, tracing the stolen bitcoins to various wallets.<br>Through their meticulous efforts, they were able to recover a significant portion of the stolen funds, providing the client with a glimmer of hope in an otherwise devastating situation. For many cryptocurrency enthusiasts, the nightmare of forgetting wallet passwords or losing access to private keys is all too real. In a heartening example, Infiniti Digital Recovery was able to help a client regain access to their forgotten wallet, unlocking a substantial amount of misplaced bitcoins. With their expertise in wallet recovery techniques and dedication to delivering results, Infiniti Digital Recovery demonstrated their ability to turn lost hopes into found fortunes. When bitcoins are stolen, it may seem like a lost cause. However, Infiniti Digital Recovery has repeatedly proven that it's never too late to recover stolen funds. In a recent case, a victim of cryptocurrency theft approached Infiniti Digital Recovery in despair. Through their advanced tracking methods and relentless pursuit, the team traced the stolen bitcoins to multiple addresses, eventually leading to the identification and apprehension of the cybercriminal. This success story highlights Infiniti Digital Recovery's commitment to justice and ability to recover even the most seemingly irretrievable funds. Reach out to them via<br><br></div><div>Email:infinitedigitalrecovery@techie.com<br>Website: infinitedigital.online</div><div class="gmail-yj6qo"></div><div class="gmail-adL">Whatsapp:<a class="gmail-gs-title" href="https://www.google.com/url?client=internal-element-cse&amp;cx=e644b33a7719ef93e&amp;q=https://www.rapidtables.com/math/symbols/Plus_Sign.html&amp;sa=U&amp;ved=2ahUKEwiTxKWnqcSMAxU1BfsDHSGBEHEQFnoECAwQAQ&amp;usg=AOvVaw0lnOzp0pwX1kAZyMvRawmO" target="_blank" dir="ltr" style="color: rgb(17, 85, 204); box-sizing: border-box; text-decoration-line: none; cursor: pointer; height: auto; overflow: hidden; font-size: 16px; font-family: arial, sans-serif;"> +</a>19085848908</div></div><div class="gmail-adL"><blockquote class="gmail_quote" style="margin: 0px 0px 0px 0.8ex; border-left: 1px solid rgb(204, 204, 204); padding-left: 1ex;"><div dir="ltr"></div></blockquote></div></div></div><div class="gmail-adL"></div></div></div><div class="gmail-WhmR8e" style="clear: both;"></div></div></div><div class="gmail-ajx" style="clear: both;"></div></div><div class="gmail-gA gmail-gt gmail-acV" style="font-size: 0.875rem; padding: 0px; width: auto; border-bottom-left-radius: 0px; border-bottom-right-radius: 0px; border-top: none; margin: 0px; background-image: initial; background-position: initial; background-size: initial; background-repeat: initial; background-attachment: initial; background-origin: initial; background-clip: initial; font-family: &quot;Google Sans&quot;, Roboto, RobotoDraft, Helvetica, Arial, sans-serif;"><div class="gmail-gB gmail-xu" style="border-top: 0px; padding: 0px;"><div class="gmail-mVCoBd"></div><div class="gmail-ip gmail-iq" style="clear: both; margin: 0px; padding: 16px 0px; border-top: none;"><div id="gmail-:g1">
  |  
-- | --



<body>
<div class="gmail-adn gmail-ads" style="border-left: none; padding: 0px; display: flex; font-family: &quot;Google Sans&quot;, Roboto, RobotoDraft, Helvetica, Arial, sans-serif; font-size: medium;"><div class="gmail-gs" style="margin: 0px; min-width: 0px; padding: 0px 0px 20px; width: auto;"><div class="gmail-"><div id="gmail-:g2" class="gmail-ii gmail-gt" style="direction: ltr; margin: 8px 0px 0px; padding: 0px; font-size: 0.875rem; overflow-x: hidden;"><div id="gmail-:g3" class="gmail-a3s gmail-aiL" style="direction: ltr; font-variant-numeric: normal; font-variant-east-asian: normal; font-variant-alternates: normal; font-size-adjust: none; font-kerning: auto; font-optical-sizing: auto; font-feature-settings: normal; font-variation-settings: normal; font-variant-position: normal; font-variant-emoji: normal; font-stretch: normal; font-size: small; line-height: 1.5; font-family: Arial, Helvetica, sans-serif; overflow: auto hidden;"><div>In one notable case, a victim of a hacked cryptocurrency exchange turned to Infinite Digital Recovery for help.<br></div><div><div class="gmail_quote"><div dir="auto"><div>Despite the complex nature of the incident, the team at Infiniti Digital Recovery diligently investigated the blockchain, tracing the stolen bitcoins to various wallets.<br>Through their meticulous efforts, they were able to recover a significant portion of the stolen funds, providing the client with a glimmer of hope in an otherwise devastating situation. For many cryptocurrency enthusiasts, the nightmare of forgetting wallet passwords or losing access to private keys is all too real. In a heartening example, Infiniti Digital Recovery was able to help a client regain access to their forgotten wallet, unlocking a substantial amount of misplaced bitcoins. With their expertise in wallet recovery techniques and dedication to delivering results, Infiniti Digital Recovery demonstrated their ability to turn lost hopes into found fortunes. When bitcoins are stolen, it may seem like a lost cause. However, Infiniti Digital Recovery has repeatedly proven that it's never too late to recover stolen funds. In a recent case, a victim of cryptocurrency theft approached Infiniti Digital Recovery in despair. Through their advanced tracking methods and relentless pursuit, the team traced the stolen bitcoins to multiple addresses, eventually leading to the identification and apprehension of the cybercriminal. This success story highlights Infiniti Digital Recovery's commitment to justice and ability to recover even the most seemingly irretrievable funds. Reach out to them via<br><br></div><div>Email:infinitedigitalrecovery@techie.com<br>Website: infinitedigital.online</div><div class="gmail-yj6qo"></div><div class="gmail-adL">Whatsapp:<a class="gmail-gs-title" href="https://www.google.com/url?client=internal-element-cse&amp;cx=e644b33a7719ef93e&amp;q=https://www.rapidtables.com/math/symbols/Plus_Sign.html&amp;sa=U&amp;ved=2ahUKEwiTxKWnqcSMAxU1BfsDHSGBEHEQFnoECAwQAQ&amp;usg=AOvVaw0lnOzp0pwX1kAZyMvRawmO" target="_blank" dir="ltr" style="color: rgb(17, 85, 204); box-sizing: border-box; text-decoration-line: none; cursor: pointer; height: auto; overflow: hidden; font-size: 16px; font-family: arial, sans-serif;"> +</a>19085848908</div></div><div class="gmail-adL"><blockquote class="gmail_quote" style="margin: 0px 0px 0px 0.8ex; border-left: 1px solid rgb(204, 204, 204); padding-left: 1ex;"><div dir="ltr"></div></blockquote></div></div></div><div class="gmail-adL"></div></div></div><div class="gmail-WhmR8e" style="clear: both;"></div></div></div><div class="gmail-ajx" style="clear: both;"></div></div><div class="gmail-gA gmail-gt gmail-acV" style="font-size: 0.875rem; padding: 0px; width: auto; border-bottom-left-radius: 0px; border-bottom-right-radius: 0px; border-top: none; margin: 0px; background-image: initial; background-position: initial; background-size: initial; background-repeat: initial; background-attachment: initial; background-origin: initial; background-clip: initial; font-family: &quot;Google Sans&quot;, Roboto, RobotoDraft, Helvetica, Arial, sans-serif;"><div class="gmail-gB gmail-xu" style="border-top: 0px; padding: 0px;"><div class="gmail-mVCoBd"></div><div class="gmail-ip gmail-iq" style="clear: both; margin: 0px; padding: 16px 0px; border-top: none;"><div id="gmail-:g1"><table class="gmail-cf gmail-wS" role="presentation" style="border-collapse: collapse;"><tbody><tr><td class="gmail-amq" style="margin: 0px; padding: 0px 16px; vertical-align: top; width: 44px; visibility: hidden;"></td><td class="gmail-amr" style="margin: 0px; padding: 0px; width: 864px;"><div class="gmail-brb" style="box-shadow: rgba(100, 121, 143, 0.12) 0px -1px 0px 0px inset; margin-bottom: 24px; padding-bottom: 16px;"><br style="color: rgb(34, 34, 34); font-family: &quot;Google Sans&quot;, Roboto, RobotoDraft, Helvetica, Arial, sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">
</body>
</html>In one notable case, a victim of a hacked cryptocurrency exchange turned to Infinite Digital Recovery for help.
Despite the complex nature of the incident, the team at Infiniti Digital Recovery diligently investigated the blockchain, tracing the stolen bitcoins to various wallets.
Through their meticulous efforts, they were able to recover a significant portion of the stolen funds, providing the client with a glimmer of hope in an otherwise devastating situation. For many cryptocurrency enthusiasts, the nightmare of forgetting wallet passwords or losing access to private keys is all too real. In a heartening example, Infiniti Digital Recovery was able to help a client regain access to their forgotten wallet, unlocking a substantial amount of misplaced bitcoins. With their expertise in wallet recovery techniques and dedication to delivering results, Infiniti Digital Recovery demonstrated their ability to turn lost hopes into found fortunes. When bitcoins are stolen, it may seem like a lost cause. However, Infiniti Digital Recovery has repeatedly proven that it's never too late to recover stolen funds. In a recent case, a victim of cryptocurrency theft approached Infiniti Digital Recovery in despair. Through their advanced tracking methods and relentless pursuit, the team traced the stolen bitcoins to multiple addresses, eventually leading to the identification and apprehension of the cybercriminal. This success story highlights Infiniti Digital Recovery's commitment to justice and ability to recover even the most seemingly irretrievable funds. Reach out to them via

Email:infinitedigitalrecovery@techie.com
Website: infinitedigital.online
Whatsapp:[ +](https://www.google.com/url?client=internal-element-cse&cx=e644b33a7719ef93e&q=https://www.rapidtables.com/math/symbols/Plus_Sign.html&sa=U&ved=2ahUKEwiTxKWnqcSMAxU1BfsDHSGBEHEQFnoECAwQAQ&usg=AOvVaw0lnOzp0pwX1kAZyMvRawmO)19085848908

**Comment by klaume68** [comment]
Recover the wallet password with the .json file

Incredible... I can't believe it... were my first thoughts when Robert wrote me! Good news!...

I was an early Ethereum adopter since the Genesis block, but I forgot the password of a wallet, and almost 8 years later, I can reaccess it with Infinite Digital Recovery. help. I had tried to recover the password for the last two years with hundreds of tests. Still, I failed and almost gave up when I read about Infinite Digital Recovery. works, I recovered my hope and decided to contact him. I handed over the .json file and some clues of the key, including Robert's ability to find the password in just 4 hours, which was amazing.
He was better for me was he? He fulfilled our contract’s terms and immediately sent me the recovered resources, so I recommend him 100%. Best regards

How to recover a Bitcoin Wallet Password- Infinite Digital Recovery.


What are wallet recovery services or crypto recovery services?

Wallet Recovery Services is a general term used by crypto recovery companies to brand themselves as a company that can help you recover funds. There are two types, one is offering password recovery, like our company, Infinite Digital Recovery. Others offer scam recovery, which is in itself a scam, as there is no way to reverse transactions on the blockchain.

What does Wallet Recovery Services do?

Most Wallet Recovery Services help their clients with the recovery of passwords to their crypto holdings. If you have created a blockchain.com account and hold a wallet ID in the format XXXX-XXXX-XXXXXXXXXXXXX-XXXX-XXXX, then it is possible to download the encrypted backup and, with the help of your best password guesses, recover the wallet password.

How much does Wallet Recovery Services charge?

The average cost to hire one of the top main services is 20% as a success fee. Any solid recovery service does not charge any upfront fees. If you are asked to pay upfront, then it's most likely a scam. There are also recovery services offering an initial lower fee, like 15% of the first week, and then they will ask for a higher fee if the wallet is not opened in the first run. There are also hardware recovery companies that can help you unlock a Trezor hardware wallet; these are generally more expensive and could charge up to 30% of the amount locked in the device.

Where can I find a trustworthy Service?

Bitcoin.com has conducted a survey and listed its most reputable Wallet Recovery Services. You can find it in Infinite Digital Recovery.

What are the most important aspects of a good and solid service?

Transparency and Accountability
Industry Leadership and Innovation
Exceptional Customer Satisfaction
Strong commitment to privacy and security
Legal Compliance and Stability
Educational resources available
Community Engagement and Collaboration

Can you recommend reading more about your work?

The Wallet Recovery Services blog is dedicated to wallet recovery. It has been maintained since 2019 and describes various recoveries and guides to successfully open a locked wallet.

How to Recover a Bitcoin Wallet Password
Dogecoin Password Recovery
Get Password Recovery
Metamask Password Recovery

If you need help recovering your Electrum Bitcoin Wallet, contact Infinite Digital Recovery  through

WEBSITE: https://infinitedigital.online/
WHATSAPP:   + 1 323 554 3592
EMAIL: infinitedigitalrecovery@techie.com

**Comment by angeldavid0576** [comment]
Best Crypto Recovery Services: Hire CERTIFIED RECOVERY SERVICES to Get Your Lost Funds Back

The moment I realized that $50,000 worth of USD Coin (USDC) had been stolen from my crypto wallet, my world shattered. As someone deeply invested in the promise of decentralized finance, the betrayal felt like more than just a financial loss; it was a profound violation of trust in the ecosystem I had championed. The scam unfolded through a sophisticated link disguised as a legitimate DeFi platform update. One click on that malicious link, and my funds vanished into an anonymous digital abyss. Panic set in immediately. I spent hours scouring blockchain explorers, tracing transactions to untraceable addresses, and pleading with exchange support teams, only to hit dead ends. The helplessness was paralyzing, and I felt as if I were trapped in a nightmare with no way out. With no other options left, I began researching crypto recovery services, though skepticism gnawed at me. How could anyone reverse irreversible blockchain transactions? Certified Recovery Services kept surfacing in forums, with users praising their forensic approach to tracking stolen assets. With nothing left to lose, I submitted a detailed case through their portal. Their response came within hours: a meticulous questionnaire about the theft timeline, wallet addresses, and transaction hashes. What followed was a masterclass in blockchain investigation. Within 48 hours, they had identified the thief’s liquidity pool withdrawals across three exchanges. Their team worked diligently, employing various strategies to trace the stolen funds. Through legal pressure and chain analysis, they successfully recovered 95% of my USDC, returning it to a newly secured wallet. The relief I felt was indescribable. If you ever find yourself a victim of a cryptocurrency scam, I highly recommend contacting Certified Recovery Services for assistance. Their expertise can make a world of difference in reclaiming lost assets and restoring your faith in the crypto ecosystem. 
Here's Their Info Below:
WhatsApp: (+1(740)258‑1417 )
Telegram: https: //t.me/certifiedrecoveryservices
mail: (certifiedrecoveryservices @zohomail .com, certified @financier .com)
Website info;( https: //certifiedrecoveryservices .com)

![Image](https://github.com/user-attachments/assets/43ebee60-06e1-4e06-9ce0-fec3cd3ef276)

**Comment by SSrett12** [comment]
HIRE SPARTAN TECH GROUP RETRIEVAL FOR CRYPTO RECOVERY CASES


Losing a secured cryptocurrency to a scam was one of the most devastating experiences of my life, but thanks to SPARTAN TECH GROUP RETRIEVAL, I was able to recover everything I lost. It all started when I fell victim to what seemed like a legitimate “free token giveaway.” The website looked incredibly professional, filled with fake endorsements, clean visuals, and a seamless user interface that made the process feel completely trustworthy. I connected my wallet and approved what appeared to be a standard transaction. Within seconds, I watched in disbelief as $53,000 disappeared from my account. It was an immediate and painful realization I had been scammed.The panic that followed was overwhelming. I felt helpless, angry, and embarrassed. I began searching online for anyone who could help, and that’s when I came across SPARTAN TECH GROUP RETRIEVAL. I reached out without much expectation, assuming the chances of recovering stolen crypto were slim to none. To my surprise, they responded quickly and professionally. Their team immediately got to work and reassured me that they would do everything possible to assist.SPARTAN TECH GROUP RETRIEVAL launched a full-scale investigation, using blockchain forensics to trace the stolen assets. They followed the funds across wallets and exchanges, identifying the scammer’s activity and tracking the flow of transactions. Their attention to detail and knowledge of how crypto scams work was impressive. They were transparent throughout the process, kept me updated at every step, and never overpromised. Still, they delivered results beyond anything I hoped for.Their persistence paid off. Through their efforts, they were able to locate the stolen assets, coordinate with the proper channels, and freeze the scammer’s accounts. The most incredible part? I recovered 100% of my stolen funds. Every dollar that was taken from me was returned. I never thought that would be possible, but they made it happen.What stood out the most was not just their technical expertise, but their empathy and commitment to helping people who have been victimized. They treated my case with urgency and care, and I truly felt like they were fighting for me.If you’ve been the victim of a crypto scam, I can’t recommend SPARTAN TECH GROUP RETRIEVAL enough. They turned my worst financial moment into a story of complete recovery and renewed trust.MORE INFO ABOUT THEM
Email:  s u p p o r t  @  s p a r t a n   t e c h  g r o u p   r e t r ieval. o r  g
Website: h  t t p s : / / spa rtan te ch group ret rieval . o r g
WhatsApp: + 1 9 7 1  4 8 7 3 5 3 8
Telegram: + 1  5 8  1  2 8 6  8 0 9 2

![Image](https://github.com/user-attachments/assets/d29b4768-dced-47a9-8c26-644c91740d26)

**Comment by marssystems** [comment]
This is a SCAM - don't even think about it !!

**Comment by dhllnjane** [comment]
WhatSapp Info: +1 4 1 4 8 0 7 1 4 8 5
Telegram Info: https: // t. me/ Rapid digital recovery519
Email Info: rapid digital recovery (@) execs. com

My 77 year-old mother, who has limited experience with technology, became a victim of a sophisticated cryptocurrency scam. Scammers impersonated Jeff Bezos, luring her with promises of substantial investment returns in exchange for Bitcoin. Trusting the offer, she transferred a significant amount of BTC valued at $17,800 only to realize later that it was a scam. The moment she understood what had happened, she was devastated. The loss weighed heavily on her, leaving her feeling embarrassed and disheartened, especially given her age and the financial implications. In her desperation, I took it upon myself to seek help. I contacted RAPID DIGITAL RECOVERY, a cybersecurity firm that specializes in investigating cryptocurrency fraud. From our very first conversation, their team exhibited remarkable professionalism and compassion. They recognized the emotional turmoil my mother was experiencing and approached her with kindness. Aware of her unfamiliarity with blockchain technology, they took the time to explain each step of the recovery process in a way that was easy for her to understand. The experts at RAPID DIGITAL RECOVERY conducted a meticulous investigation, employing advanced techniques to trace the stolen Bitcoin. They ensured that my mother felt comfortable and informed throughout the entire process, which was crucial in helping her regain some sense of control. Their dedication and transparency provided her with hope during a time when she felt lost. After a week of hard work, the team successfully traced a portion of the stolen funds. They managed to recover $15,000 worth of Bitcoin, which brought immense relief to my mother. This recovery not only eased her financial burden but also restored her faith in the possibility of justice. She expressed her gratitude, saying, “I never thought I would see any of that money again. I’m so thankful for RAPID DIGITAL RECOVERY help. My mother was able to recover everything, and I felt a deep sense of pride in being the one who sought help from RAPID DIGITAL RECOVERY.
